opencdms/models/cdm.py

```diff
@@ -30,7 +30,7 @@
 Geography = NewType("Geography", str)
```

Comment:
Delete this line and  import Geography from geoalchemy2

Reply:
I think you were correct to not import from geoalchemy2 in `cdm.py` since in the domain model there shouldn't have any dependencies on the database layer.